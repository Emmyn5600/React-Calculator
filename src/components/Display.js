/* eslint-disable react/prop-types */
import React from 'react';

const Display = ({ calculation }) => (
  <div className="display-result">

    <input id="result" defaultValue={calculation} placeholder="0" dir="rtl" />

  </div>

);

export default Display;
