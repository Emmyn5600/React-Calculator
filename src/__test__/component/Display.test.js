import React from 'react';
import renderer from 'react-test-renderer';
import Display from '../../components/Display';

describe('Display', () => {
    it('it renders the display', () => {
     const tree = renderer.create(<Display/>).toJSON();
     expect(tree).toMatchSnapshot();
    });
});