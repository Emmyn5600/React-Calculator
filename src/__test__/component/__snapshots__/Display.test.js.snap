// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display it render the display 1`] = `
<div
  className="display-result"
>
  <input
    dir="rtl"
    id="result"
    placeholder="0"
  />
</div>
`;

exports[`Display it renders the display 1`] = `
<div
  className="display-result"
>
  <input
    dir="rtl"
    id="result"
    placeholder="0"
  />
</div>
`;
